styles/style.styl

```diff
@@ -1,9 +1,13 @@
-
-form 
-    padding: 20px;
+fontSizeMedium = 12px;
 
 .mybutton
```

Comment:
In future PR, change that class name - it doesn't seem to describe what the class isd oing at all. Also, consider adding some general-purpose classes for spacing you can use on any element, not only on a button (i.e.: like `.margin-top-medium` or `.mtm` etc)

Reply:
https://github.com/EwaTrzemzalska/zjemto/issues/34